model/otel/metrics.yaml

```diff
@@ -0,0 +1,105 @@
+groups:
+  - id: metric.otel.sdk.span.created_count
+    type: metric
+    metric_name: otel.sdk.span.created_count
+    stability: development
+    brief: "The number of spans which have been created"
+    instrument: counter
+    unit: "{span}"
+    attributes:
+      - ref: otel.span.is_sampled
+
+  - id: metric.otel.sdk.span.live_count
+    type: metric
+    metric_name: otel.sdk.span.live_count
+    stability: development
+    brief: "The number of created spans for which the end operation has not been called yet"
+    instrument: updowncounter
+    unit: "{span}"
+    attributes:
+      - ref: otel.span.is_sampled
+
+  - id: metric.otel.sdk.span.ended_count
+    type: metric
+    metric_name: otel.sdk.span.ended_count
+    stability: development
+    brief: "The number of created spans for which the end operation was called"
+    instrument: counter
+    unit: "{span}"
+    attributes:
+      - ref: otel.span.is_sampled
+
+
+  - id: metric.otel.sdk.span.processor.queue_size
+    type: metric
+    metric_name: otel.sdk.span.processor.queue_size
+    stability: development
+    brief: "The number of spans in the queue of a given instance of an SDK span processor"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: updowncounter
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
+      - ref: otel.sdk.component.name
+
+  - id: metric.otel.sdk.span.processor.queue_capacity
+    type: metric
+    metric_name: otel.sdk.span.processor.queue_capacity
+    stability: development
+    brief: "The maximum number of spans the queue of a given instance of an SDK span processor can hold"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: gauge
```

Comment:
I think this should be an updowncounter.  I would expect capacities to be added, rather than averaged when aggregated. See https://github.com/open-telemetry/semantic-conventions/blob/ecb9aa011f3c1496bf16cf15371a725947cf1f64/docs/system/hardware-metrics.md?plain=1#L208 for example